CHANGELOG.rst

```diff
@@ -12,6 +12,14 @@ Internal Changes
   secret defining them already exists, it will be maintained, if it does not a set of random passwords will
   be generated.
 
+- **Redis**: Now the password for Redis is optional to define in the values file: if the secret defining it already
```

Comment:
These two regard the previous PR where I forgot to mention that in the CHANGELOG